.dev/get_urls.R

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/Rscript
+# Script to get URL keys from CRAN
+#   of packags that Suggest or Import lintr
+
+cran_db <- as.data.frame(available.packages(), stringsAsFactors = FALSE)
+lintr_re <- "\\blintr\\b"
+lintr_pkg <- cran_db[
+  with(cran_db, grepl(lintr_re, Suggests) | grepl(lintr_re, Imports)),
+  "Package"
+]
+
+# simple retry mechanism
+desc_url_fmt <- "https://cran.r-project.org/web/packages/%s/DESCRIPTION"
+extract_url <- function(pkg, initial_sleep = 1, retry_sleep = 60) {
+  desc_url <- sprintf(desc_url_fmt, pkg)
+  retry <- function(cond) {
+    message("Sleepy... 😴")
+    Sys.sleep(retry_sleep)
+    url(desc_url, open = "r")
+  }
+  Sys.sleep(initial_sleep)
+  url_conn <- tryCatch(url(desc_url, open = "r"), error = retry, warning = retry)
+  pkg_data <- read.dcf(url_conn, "URL")
+  drop(pkg_data)
```

Comment:
If you do `c("URL", "BugReports")` and then `paste(drop(pkg_data), collapse = "\n")`, you'll find a bunch more packages.

Reply:
right but for the next step well also have to post-process the URLs 😩

but ok you convinced me. it's only a few extra lines.

Reply:
Postprocessing works without modifications because the /issues path isn't part of the match :)

Reply:
true... i guess there's an assumption that if both contain a github URL, they're the same, which is acceptable

Reply:
Yep (or, at least, the one listed under URL is the correct one)